### ISSUE TITLE
RFE: Facility for Drag and drop the selected text

### ISSUE THREAD

**Issue by dtrehas** [sumatrapdfreader/sumatrapdf#507]
Dear Krzysztof Kowalczyk,
thank you for bringing a very good free pdf viewer.

From my side I would be more greatful if could not only select and copy pdf text but also to 
select it and drag' n'drop it.

**Comment by GitHubRulesOK** [comment]
@kjk 
this enhanced feature request seem to be "out of scope" since to **"drop" text** into another app effectively requires external control of windows such as manipulating clip-board functionality.

The underlying requirement to be able to **select text and "pass" it** to external apps is well covered by other open issues.

**Comment by kjk** [comment]
I think it's possibly with standard drag&drop functionality in Windows. I can select text in Chrome and drop it on e.g. Visual Studio.

**Comment by SumatraPeter** [comment]
@kjk: Yeah, I think it involves implementing the proper drop source/target code as required. Not sure how much work would be required, or whether the benefit would be worth the effort...
